src/sentry/integrations/github/client.py

```diff
@@ -63,12 +68,20 @@ def get_repo(self, repo: str) -> JSONData:
         return repository
 
     def get_repositories(self) -> Sequence[JSONData]:
+        """
+        This fetches all repositories accessible to the Github App
+        https://docs.github.com/en/rest/apps/installations#list-repositories-accessible-to-the-app-installation
+        """
         # Explicitly typing to satisfy mypy.
-        repositories: JSONData = self.get("/installation/repositories", params={"per_page": 100})
-        repos = repositories["repositories"]
+        repos: JSONData = self.get_with_pagination(
```

Comment:
This is the core of the change. We're now going to use pagination and get more than 100 repos. I have no idea how this worked in the past or no one complained.

Reply:
If you remove the `per_page` param do you get 30 results now instead of 100?

Reply:
Correct. The default of their endpoint is 30 results. The max is 100.